sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -392,6 +392,12 @@ internal sealed class KnownType
 
         private readonly bool isSpecialType;
         private readonly SpecialType specialType;
+
+        static internal KnownType CreateKnownTypeForTesting(string typeName)

```

Comment:
I was going to delete this because in the end I didn't need it for the tests I've created.